backend/homework/11/src/main/java/com/example/jdbc/controller/TenantController.java

```diff
@@ -0,0 +1,129 @@
+package com.example.jdbc.controller;
+
+
+import com.example.jdbc.dto.*;
+import com.example.jdbc.model.Shift;
+import com.example.jdbc.model.ShiftType;
+import com.example.jdbc.model.ShiftUser;
+import com.example.jdbc.model.User;
+import com.example.jdbc.service.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+public class TenantController {
+    private final UserService userService;
+    private final ShiftTypeService shiftTypeService;
+    private final ShiftUserService shiftUserService;
+    private final ShiftService shiftService;
+    private final AllService allService;
+    private final TenantService tenantService;
+
+    @Autowired
+    public TenantController(
+            UserService userService,
+            ShiftTypeService shiftTypeService,
+            ShiftUserService shiftUserService,
+            ShiftService shiftService,
+            AllService allService,
+            TenantService tenantService) {
+        this.userService = userService;
+        this.shiftTypeService = shiftTypeService;
+        this.shiftUserService = shiftUserService;
+        this.shiftService = shiftService;
+        this.allService = allService;
+        this.tenantService = tenantService;
+    }
+
+    @GetMapping("/users/all")
+    public ResponseEntity<List<User>> getAllUsers() {
+        try {
+            List<User> users = userService.getAllUser();
+            return ResponseEntity.ok(users);
+        } catch (Exception e) {
+            e.printStackTrace();
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
```

Comment:
Never throw 500 as a dev, this code is something that is thrown by server internally